public/app/plugins/datasource/testdata/dataquery.cue

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grafanaplugin
+
+import (
+	"github.com/grafana/grafana/packages/grafana-schema/src/common"
+	"github.com/grafana/grafana/pkg/plugins/pfs"
+)
+
+// This file (with its sibling .cue files) implements pfs.GrafanaPlugin
+pfs.GrafanaPlugin
+
+composableKinds: DataQuery: {
+	maturity: "merged"
+
+	lineage: {
+		seqs: [
+			{
+				schemas: [
+					{
+						common.DataQuery
+						alias?:       string
+						scenarioId?:  string
+						stringInput?: string
+						stream?:      #StreamingQuery
+						pulseWave?:   #PulseWaveQuery
+						sim?:         #SimulationQuery
+						csvWave?: [...#CSVWave] //TODO can we prevent partial from being generated
```

Comment:
can we prevent partial from being generated? In this case we don't have any defaults defined. Seems like arrays always have a default value of empty?

Reply:
Yes, these should be omitted. It's a known issue in cuetsy

/me adds it to list of issues to actually post